From 66dbbaebfb2311f36c3db9d9cdcbc17e48118593 Mon Sep 17 00:00:00 2001
From: "guoping.liu" <guoping.liu@3reality.com>
Date: Tue, 9 Sep 2025 17:53:41 +0800
Subject: [PATCH] Remove shutdown

---
 drivers/gpu/drm/meson/meson_drv.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/meson/meson_drv.c b/drivers/gpu/drm/meson/meson_drv.c
index 2df2315b0..c7b6f22da 100644
--- a/drivers/gpu/drm/meson/meson_drv.c
+++ b/drivers/gpu/drm/meson/meson_drv.c
@@ -567,7 +567,6 @@ static const struct dev_pm_ops meson_drv_pm_ops = {
 static struct platform_driver meson_drm_platform_driver = {
 	.probe      = meson_drv_probe,
 	.remove     = meson_drv_remove,
-	.shutdown   = meson_drv_shutdown,
 	.driver     = {
 		.name	= "meson-drm",
 		.of_match_table = dt_match,
-- 
2.39.5

